tests/nocover/test_descriptortests.py

```diff
@@ -143,11 +143,6 @@ def integers_from(x):
     .flatmap(integers_from)
 )
 
-TestIntStreams = strategy_test_suite(streaming(integers()))
-TestStreamLists = strategy_test_suite(streaming(integers()))
-TestIntStreamStreams = strategy_test_suite(
-    streaming(streaming(integers())))
-
```

Comment:
They're generic tests that used to be useful and should probably just be ditched entirely. I figured it was easier to just remove the streaming tests (especially because they're particularly useless instances of this genre) than fix them to deal with the deprecation warning.

They date back to the days when strategies contained a *lot* more logic of their own so there were a lot of things you could generically test. That is no longer the case, and everything they test is now either a test of the engine or needs a more specific test for the file.

In fact, they should probably be removed altogether. I'll submit a followup pull request that just deletes the file and the support code for it (which is in src so will be a version bump, but I'm inclined to resist the temptation to roll it into this PR).